src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/CourtCaseEntity.java

```diff
@@ -75,7 +76,7 @@
 
     @JsonManagedReference
     @OneToMany(mappedBy = "courtCase", fetch = FetchType.EAGER, cascade = { CascadeType.ALL }, orphanRemoval=true)
-    private List<OffenceEntity> offences = new ArrayList<>();
+    private List<OffenceEntity> offences;
```

Comment:
👍  I assume this is to prevent that compiler warning, good work - I've been meaning to do the same but didn't want to risk breaking stuff 😄

Reply:
Yes